### DISCUSSION TITLE
YouTube Music 256k & 320k Released

### DISCUSSION THREAD

**Discussion by TEOL85** [aclap-dev/video-downloadhelper#425] [Help and Q&A]
### Your question

Just thought I'd give a heads up as I've been getting hits on my Audio extension because obviously the next step is to bring it into VDH so I wouldn't need to go to a different software.

For the last 2 weeks I've been getting hits for free mode videos with 256 and 320 in them particullally from 2024 Sony M uploads so I assume sound quality isn't a YouTube paid extra anymore but at the same time it means VDH has to become busy adding it in.

That means that just ordanary video on regular Video YouTube also got the upgrade as well and sound quality isn't a paid upgrade anymore for video too just that I found it on one off those audios with a video picture. 

But then again maybe there was a deeper soud upgrade which meant 256 & 320 Vorbis can come to us like m4a getting 448 dolby in the paid mode which meant 256 and 320 vorbis in stereo were free to go to everyone now in Q2 2024 who knows but it's exciting we are getting new audio qualitys coming for free video now it was long overdue as 192 and 256 shouldn't have been took away but to get 320 now was a surprise matches the highest mp3 bitrate you can get. 

Edit: In stats for nerds in a video that I know had the 320 these were the stats for nerds on 1080 HD.

vp09.00.51.08.01.01.01.01.00 (248) / opus (251)

Well I can't actaully find 256 anymore so that might be scrapped for only 320k in the higher audios. Thats alright by me though the better it is the better it is and would be great if VDH gets it fast. 

And of course a link that supports it might help as well as the version of it as written on the other software which has it downloading.

https://youtu.be/XGVnfG6QuSA?si=MoJ5rl8T1J-ImeDo

https://rr5---sn-cu-aigsl.googlevideo.com/videoplayback?expire=1713976737&ei=QeEoZsRMvq2n6g-OiIzgDw&ip=81.157.132.201&id=o-AJL9N6-lo8gkbC0SljY3NYpErjfL7ZNQ6OkGet3c9LiH&itag=774&source=youtube&requiressl=yes&xpc=EgVo2aDSNQ%3D%3D&mh=O9&mm=31%2C29&mn=sn-cu-aigsl%2Csn-cu-c9i6&ms=au%2Crdu&mv=m&mvi=5&pl=25&gcr=gb&initcwndbps=1481250&siu=1&bui=AWRWj2QYseRNF4gADbA38J-3BfVO5iMx8QIxK1GEV8sOZFLNRjXzfC4TjmH-MjT2_ExVs9tdFA&spc=UWF9f0AWK6a2MhCN3NEcTf6qE4JkytWudwCVzKctYG1JRr0dN8YHoNqRsZaEmKpcMFSFmAG8qg&vprv=1&svpuc=1&mime=audio%2Fwebm&ns=5KDWwt-_pub5mXEdIfmekSwQ&gir=yes&clen=5996951&dur=176.621&lmt=1713377431242425&mt=1713954530&fvip=3&keepalive=yes&c=WEB&sefc=1&txp=2318224&n=4wBhIzdnS5R_SGv&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cxpc%2Cgcr%2Csiu%2Cbui%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cns%2Cgir%2Cclen%2Cdur%2Clmt&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AHWaYeowRAIgYvGfRExhqG1TtcXYzO5xAxqwv2G3eO4Es1dFqRfVancCIHNZFw87xjDWfLnNMVZ8rKZ48ZTHXUK59S95CKPQtIge&ratebypass=yes&sig=AJfQdSswRAIgAqfJSXbbwC4sJOWNY6HSpfCycpmfu4MU4AAlpofHyRACIDx4GNIraNAA4DmBfTF11nB-efaaGWUp0S81V2zy7Vun

Number 90 on the Heritage Artists Devision of the thing I do for fun that every artist to land on the date qualifys for a new record in the future but it supports the 320 thats the purpose.

I've found the 256k on another track in the silly chart thing I do for fun. Sames stats for nerds on the vorbis part which is weird for a different 1080p sound format code if the format itself is different but number 57 has 256k.

https://youtu.be/-FDRxtcRiEo?si=JL-J3qo4ZuXv-C2I

https://rr2---sn-cu-aigsl.googlevideo.com/videoplayback?expire=1713980295&ei=J-8oZuvjNeT2xN8P_4SQkA0&ip=81.157.132.201&id=o-AHYv9IOF-sNR5K-Ah8k2MhFr-XzZSBodnbL7Zdj2SiCO&itag=774&source=youtube&requiressl=yes&xpc=EgVo2aDSNQ%3D%3D&mh=bP&mm=31%2C29&mn=sn-cu-aigsl%2Csn-cu-c9i6&ms=au%2Crdu&mv=m&mvi=2&pl=25&gcr=gb&initcwndbps=1488750&siu=1&bui=AWRWj2RlfDdF0sfWsh_Y2Kvash19R7B9l0E1hUylvlJdcFuW2t0c4CUqn-m2jEfcfaH8gIb8Ow&spc=UWF9f4BmrQvrf-zTiNApREpam_eZroSCFjXRplz_0UZgM2cE8TI60Aq9IyqZ2CdeqNDOoeKciQ&vprv=1&svpuc=1&mime=audio%2Fwebm&ns=P7Qbi8WHdGq-A15Xoa0PeWAQ&gir=yes&clen=6888969&dur=212.961&lmt=1713383087505411&mt=1713958363&fvip=1&keepalive=yes&c=WEB&sefc=1&txp=2318224&n=dgMvtE94vWWaY4i&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cxpc%2Cgcr%2Csiu%2Cbui%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cns%2Cgir%2Cclen%2Cdur%2Clmt&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AHWaYeowRgIhAIkpRNwngZxFTciKecRuXu-ev0fYnUonXASRkQ3I7mD-AiEAynLw0-YFRpbJX0NHJC5_sj82nnDrziHWTBJywMbONzI%3D&ratebypass=yes&sig=AJfQdSswRgIhALPtjVcn0E_oocM_LAyXuzq80KQwnQC-5EX9uhbE9AGqAiEA5swpCaphcTLaVm25A2LnHhBKujs-k6DqHU2f4__CEBM%3D

I guess we now have sources to get the new formats VDH ready which is good.

### Addon & CoApp version and details

Heads up about a new format we need not really asking for help so don't find this bit relevent today.

**Comment by ksze** [comment]
Can you rewrite the above like a normal person, with proper punctuation and sentence boundaries?
